### PR TITLE
Dockerized

### PR DESCRIPTION
**Pull request by againer**
- [CORE-826](https://cloudapp.atlassian.net/browse/CORE-826?filter=11800)
- Add guardian to elastic beanstalk via docker
- not building off of staging; original `guardian` only had a production environment

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+sudo: required
+
+language: ruby
+
+services:
+  - docker
+
+before_install:
+  - docker build -t tigersquad/guardian .
+  - docker run tigersquad/guardian /bin/sh -c "echo hi"
```
Comment (dfischer) [inline .travis.yml]: are there any tests for guardian now? do they pass? https://github.com/cloudapp/guardian/tree/staging/spec

File: Dockerrun.aws.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        "sourcePath": "/var/app/current"
+      }
+    }
+  ],
+  "containerDefinitions": [
+    {
+      "name": "guardian",
+      "image": "tigersquad/guardian:0.0.1",
+      "essential": true,
+      "memory": 1400,
```
Comment (dfischer) [inline Dockerrun.aws.json]: what size instance is this deployed on?

Comment (dfischer) [inline Dockerrun.aws.json]: why 1400 memory then?

Comment (againer) [inline Dockerrun.aws.json]: no good reason; copied it over and forgot to change it

**Pull request merged**